.github/workflows/styler-actions.yml

```
@@ -0,0 +1,78 @@
+on:
+  issue_comment:
+    types: [created]
+name: Commands
+jobs:
+  style:
+    if: startsWith(github.event.comment.body, '/style')
+    name: style
+    runs-on: macOS-latest
+    steps:
+      - id: file_changes
+        uses: trilom/file-changes-action@v1.2.3
+      - name: testing
+        run: echo '${{ steps.file_changes.outputs.files_modified}}'   
+      - uses: actions/checkout@v2
+      - uses: r-lib/actions/pr-fetch@master
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+      - uses: r-lib/actions/setup-r@master
+      - name: Install dependencies
+        run: |
+          Rscript -e 'install.packages(c("styler", "devtools"), repos = "cloud.r-project.org")'
+          Rscript -e 'devtools::install_version("roxygen2", version = "7.0.2", repos = "http://cran.us.r-project.org")'
+      - name: string operations
+        run: |
+          echo '${{ steps.file_changes.outputs.files_modified}}' > names.txt
+          cat names.txt | tr -d '[]' > changed_files.txt
+          text=$(cat changed_files.txt)
+          IFS=',' read -ra ids <<< "$text"
+          for i in "${ids[@]}"; do if [ "$i" == *.R\" || "$i" == *.Rmd\" ]; then echo "$i" >> files_to_style.txt; fi; done
```

```suggestion
        shell: bash
        run: |
          echo '${{ steps.file_changes.outputs.files_modified}}' > names.txt
          cat names.txt | tr -d '[]' > changed_files.txt
          text=$(cat changed_files.txt)
          IFS=',' read -ra ids <<< "$text"
          for i in "${ids[@]}"; do if [[ "$i" == *.R\" || "$i" == *.Rmd\" ]]; then echo "$i" >> files_to_style.txt; fi; done
```

My mistake when I suggested single-brackets here was forgetting that || doesn't work inside them. We could go fully down the POSIX rabbit hole and write `... do test "$i" = *.R\" -o "$i" = *.Rmd\" && echo "$i" >> ...`, but I think the bash version is way more readable.